src/classes/BDI_DataImportService.cls

```diff
@@ -41,6 +41,9 @@ global with sharing class BDI_DataImportService {
     public static final String FM_HELP_TEXT = 'Help Text';
     public static final String FM_DATA_IMPORT_FIELD_MAPPING = 'Data Import Field Mapping';
 
+    @TestVisible
+    private static final String PAYMENT_SERVICES_SUCCESSFUL_CHARGE = 'captured';
```

Comment:
I think we should probably define the entire enumerated vocab for the payment status somewhere - and I think that probably belongs somewhere else (maybe GE_PaymentServices?). I think something more similar to this: 
`    /***
    * @description AsyncApexJob possible status values
    */
    public enum ApexJobStatus {
        HOLDING, QUEUED, PREPARING, PROCESSING, COMPLETED, FAILED, ABORTED
    }

    /***
    * @description AsyncApexJob possible status values indicating the batch job is in progress
    */
    public static final Set<String> IN_PROGRESS_STATUSES = new Set<String>{
        ApexJobStatus.HOLDING.name(),
        ApexJobStatus.QUEUED.name(),
        ApexJobStatus.PREPARING.name(),
        ApexJobStatus.PROCESSING.name()
    };`

Reply:
(I don't know if we need to build that set of strings, but might have to use the .name() to compare the string value of the field with the enumeration we define)

Reply:
We have the enum in javascript in geGiftEntryFormApp.js as 
const PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze({
    PENDING: 'PENDING',
    AUTHORIZED: 'AUTHORIZED',
    CANCELED: 'CANCELED',
    CAPTURED: 'CAPTURED',
    DECLINED: 'DECLINED',
    NONRETRYABLEERROR: 'NONRETRYABLEERROR',
    RETRYABLEERROR: 'RETRYABLEERROR',
    REFUNDISSUED: 'REFUNDISSUED'
});
It would be great to store them in only one place (apex) and import them in js.

Reply:
@vmartinov-sforg @ptewson-sfdo yeah we can do that. I also think `GE_PaymentServices` would be a good candidate for that since we have defined that class as containing all PS related methods and functionality.